python/ray/train/examples/train_linear_dataset_example.py

```diff
@@ -20,10 +20,8 @@ def get_dataset(a, b, size) -> Dataset:
 
     dataset = get_dataset(a, b, size)
 
-    split_index = int(dataset.count() * split)
-
-    train_dataset, validation_dataset = dataset.random_shuffle().split_at_indices(
-        [split_index]
+    train_dataset, validation_dataset = dataset.random_shuffle().split_proportionately(
```

Comment:
Use train_test_split here?

Reply:
This is not an AIR example, so I didn't want to use it there

Reply:
Ah oops- did not see that.